deployments/helm/nsm/charts/spire/templates/agent-configmap.tpl

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-agent
+  namespace: {{ .Release.Namespace }}
+data:
+  agent.conf: |
+    agent {
+      data_dir = "/run/spire"
+      log_level = "DEBUG"
+      server_address = "spire-server"
+      server_port = "8081"
+      socket_path = "/run/spire/sockets/agent.sock"
+      trust_bundle_path = "/run/spire/config/bootstrap.crt"
+      trust_domain = "test.com"
+    }
+    plugins {
+      NodeAttestor "k8s_sat" {
+        plugin_data {
+          # NOTE: Change this to your cluster name
+          cluster = "kubernetes"
+        }
+      }
+      KeyManager "memory" {
+        plugin_data {
+        }
+      }
+      WorkloadAttestor "k8s" {
+        plugin_data {
+          {{- if .Values.azure }}
+          kubelet_read_only_port = 10255
+          {{- else }}
+          skip_kubelet_verification = true
+          {{- end }}
+        }
+      }
+    }
+  bootstrap.crt: |
```

Comment:
Why are we checking a bootstrap cert into git?

Reply:
For the first time, to avoid complexity of quite a big PR. There are related comment from Andrey https://github.com/networkservicemesh/networkservicemesh/pull/1380#discussion_r314268283

Reply:
Got it, could we get an issue filed and comment here (and other places with embedded certs) pointing to that issue so if folks come across it in the interim they know we aren't just wildly insecure :)

Reply:
For the first time, to avoid complexity of quite a big PR. There are related comment from Andrey https://github.com/networkservicemesh/networkservicemesh/pull/1380#discussion_r314268283

Reply:
> Got it, could we get an issue filed and comment here (and other places with embedded certs) pointing to that issue so if folks come across it in the interim they know we aren't just wildly insecure :)

Sure!